build/Dockerfile

```diff
@@ -36,4 +36,4 @@ COPY server/. /workspace/server/
 
 # Run server
 WORKDIR /workspace/server
-CMD if [ "$ENV" = "local" ] ; then python3.7 main.py ; else exec gunicorn --bind :8080 main:app ; fi
\ No newline at end of file
+CMD if [ "$ENV" = "local" ] ; then python3.7 main.py ; else exec gunicorn --timeout 60 --bind :8080 main:app ; fi
```

Comment:
for my understanding, is the 60s in the yaml for prod and this one for local/dev?

Reply:
This is for Kubernetes deployment, ie when "ENV" != "local". So this applies to prod/dev

Local instance will use "python3.7 main.py" which does not have a timeout.